### PR TITLE
[MM-32708] Add documentation for the jitsi and monitoring Omnibus components

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
This PR adds docs for the external components in Omnibus

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-32708

### CODE REVIEW

**Review by justinegeffen — COMMENTED** [review]
Awesome, thanks @mgdelacroix! I've done a bit of rewording here and there - let me know if anything is questionable. :)

**Comment by mgdelacroix** [comment]
> Awesome, thanks @mgdelacroix! I've done a bit of rewording here and there - let me know if anything is questionable. :)

wow, way more homogeneous and polished, thanks **a ton** @justinegeffen!!!

File: source/install/mattermost-omnibus.rst [hunk 1]
```diff
@@ -65,13 +65,23 @@ The properties that you can configure in this file are:
...
+- ``monitoring_installed``: This setting can be ``true`` or ``false`` and is used to install the monitoring component.
+- ``monitoring_enabled``: This setting can be ``true`` or ``false`` and is used to enable the monitoring component.
+- ``grafana_user``: The monitoring component Grafana username. This setting should be set if the monitoring component is enabled.
+- ``grafana_password``: The monitoring component Grafana password. This setting should be set if the monitoring component is enabled.
+- ``jitsi_installed``: This setting can be ``true`` or ``false`` and is used to install the Jitsi component.
+- ``jitsi_enabled``: This setting can be ``true`` or ``false`` and is used to enable the Jitsi component.
+- ``jitsi_fqdn``: The domain name of the Jitsi component.
+- ``jitsi_jvb_secret``: An internal Jitsi secret used to communicate between its prosody and video-bridge modules.
+- ``jitsi_focus_secret``: An internal Jitsi secret used to communicate between its prosody and jicofo modules.
+- ``jitsi_focus_password``: An internal Jitsi secret used to connect jicofo and prosody.
```
Comment (jespino) [inline source/install/mattermost-omnibus.rst:77]: Maybe we can rewrite it in a more explanatory way, something like `The password for the "focus" user in prosody. This is used to connect jicofo to prosody.`

File: source/install/mattermost-omnibus.rst [hunk 2]
```diff
@@ -81,6 +91,44 @@ Please be careful when using this feature, as making changes to the custom templ
...
+With all the values set, run ``sudo mmomni reconfigure`` for Omnibus to apply the changes and restart Mattermost. When the reconfigure process is finished, use ``jitsi_fqdn`` to access your new Jitsi instance.
+
+To integrate Jitsi and Mattermost, you can install and configure the `Mattermost Jitsi plugin <https://github.com/mattermost/mattermost-plugin-jitsi>`_.
+
+To uninstall the Jitsi component, open the ``mmomni.yml`` configuration file and set ``jitsi_installed`` and ``jitsi_enabled`` to ``false``. Then run ``sudo mmomni reconfigure`` for Omnibus to apply the changes and restart Mattermost.
+
+Monitoring
+^^^^^^^^^^
+
+This component installs and configures a `Grafana <https://grafana.com/>`_ instance with a set of default dashboards that monitor the health of your Mattermost Omnibus platform.
```
Comment (jespino) [inline source/install/mattermost-omnibus.rst:121]: This is not about grafana, this is about the Mattermost monitoring system. that includes prometheus and grafana, as the main pieces.

Besides, I would clearly point here, that this component is only usable with E20 license.

**Review by jespino — CHANGES_REQUESTED** [review]
A couple of details.

File: source/install/mattermost-omnibus.rst (hunk 1 above)
Comment (mgdelacroix) [reply source/install/mattermost-omnibus.rst]: Addressed in 140e755

File: source/install/mattermost-omnibus.rst (hunk 2 above)
Comment (mgdelacroix) [reply source/install/mattermost-omnibus.rst]: Addressed in 03dfb61

**Comment by justinegeffen** [comment]
No problem, @mgdelacroix! It's easy to work with solid foundational content. :) Glad you're happy with the suggestions. :)

**Review by justinegeffen — APPROVED** [review]

**Review by jespino — APPROVED** [review]
LGTM

**Review by wiersgallak — APPROVED** [review]

**Comment by cwarnermm** [comment]
@wiersgallak - Do you know what release this update is waiting on?

**Comment by wiersgallak** [comment]
@cwarnermm I do not.  

@mgdelacroix Sorry if we dropped the ball on this.  Do you know when these features were added or are expected to be added?

**Comment by mgdelacroix** [comment]
@wiersgallak @cwarnermm apologies for the late reply. This docs are waiting on https://github.com/mattermost/mattermost-omnibus/pull/11, which got some issues detected on testing and is waiting for the fixes to be retested and merged

//cc @jespino

**Comment by cwarnermm** [comment]
@mgdelacroix - Given that https://github.com/mattermost/mattermost-omnibus/pull/11 is in a draft state, and given the age of this PR, does it make sense to keep this PR open, or close it from your perspective?

**Comment by mgdelacroix** [comment]
@cwarnermm I think it makes more sense to close this one and reopen it if/when https://github.com/mattermost/mattermost-omnibus/pull/11 is ready. Closing it then

**Pull request closed without merging**